src/CSFML/System/Time.cpp

```diff
@@ -63,7 +63,7 @@ sfTime sfSeconds(float amount)
 ////////////////////////////////////////////////////////////
 sfTime sfMilliseconds(int32_t amount)
 {
-    return {static_cast<int64_t>(amount * 1000)};
+    return {static_cast<int64_t>(amount) * 1000};
```

Comment:
```suggestion
    return {int64_t{amount} * 1000};
```
Because this is simply a promotion, we can avoid the static cast.